brave-clean-slate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script removes Brave browser configuration, bookmarks, extensions and residual files.
+# Usage: ./brave-clean-slate.sh
+
+sudo rm -rf /Library/Application\ Support/BraveSoftware
+rm -rf ~/Library/Application\ Support/BraveSoftware
+
+# Residual files
+rm -rf /Library/Saved Application State/com.brave.Browser.savedState
```

Comment:
Thank you. Should there be a `~` in front of this?

Reply:
It would be nice to also delete `com.brave.Browser.beta.savedState` and `com.brave.Browser.nightly.savedState`.